azurerm/data_source_dns_zone.go

```diff
@@ -43,6 +43,29 @@ func dataSourceArmDnsZone() *schema.Resource {
 				Set:      schema.HashString,
 			},
 
+			"zone_type": {
+				Type:     schema.TypeString,
+				Default:  nil,
+				Optional: true,
+				Computed: true,
+			},
+
+			"registration_virtual_networks": {
```

Comment:
since this is a list of ID's - I think this'd be better as `registration_virtual_network_ids`

Reply:
good suggestion.

Reply:
done.